server/controllers/paypalClient.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-restricted-globals */
+/* eslint-disable linebreak-style */
+const checkoutNodeJssdk = require('@paypal/checkout-server-sdk');
+
+function environment() {
+  const clientId = process.env.PAYPAL_CLIENT_ID || 'AY7O6M0NDbBh3f6eaRpynKmm5v7KUgf6pWaKXJIr3UY0i10x5uPB9a6CmjUWlWD-jpZ8HWXJFuJq03fL';
+  const clientSecret = process.env.PAYPAL_CLIENT_SECRET || 'EARZHbTGFb5tyUMQG_Y5O6ScxbOeD13uxUqmOHYuylnekbR1Y8SQzH_un2XyIoU9TyxwR5k_BrEswuPF';
+
+  return new checkoutNodeJssdk.core.SandboxEnvironment(
+    clientId, clientSecret,
+  );
+}
+
+function client() {
+  return new checkoutNodeJssdk.core.PayPalHttpClient(environment());
+}
+
+async function prettyPrint(jsonData, pre = '') {
```

Comment:
could you add a citation to the paypal website where you got this

Reply:
Got it.